packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -1,8 +1,10 @@
 import { expect, haveResourceLike } from '@aws-cdk/assert';
+import { Certificate } from '@aws-cdk/aws-certificatemanager';
 import * as cdk from '@aws-cdk/core';
 import { Test } from 'nodeunit';
 
 import * as appmesh from '../lib';
+import { TlsMode } from '../lib/tls-certificate';
```

Comment:
Same import comments as above test (no per-member imports outside your module, use the qualified import).